<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QMainWindow" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="menu">
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>190</y>
        <width>281</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QPushButton" name="jouer">
         <property name="text">
          <string>JOUER</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>MENU</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="gerer">
         <property name="text">
          <string>GERER</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="jouer_2">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>542</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="7" column="5">
        <widget class="QProgressBar" name="progressBar_4">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="value">
          <number>386</number>
         </property>
         <property name="format">
          <string>%v/%m PV</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3" rowspan="8">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QProgressBar" name="progressBar_3">
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="value">
          <number>172</number>
         </property>
         <property name="format">
          <string>%v/%m PV</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QProgressBar" name="progressBar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="value">
          <number>386</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
         <property name="format">
          <string>%v/%m PV</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="4">
        <widget class="QPushButton" name="pushButton_13">
         <property name="text">
          <string>ATTAQUE</string>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QPushButton" name="menu_3">
         <property name="text">
          <string>Menu</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButton_12">
         <property name="text">
          <string>ATTAQUE</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>80</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="5">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="menu_2">
         <property name="text">
          <string>Menu</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="5" rowspan="2">
        <widget class="QLabel" name="test_img">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QProgressBar" name="progressBar_2">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
         <property name="value">
          <number>172</number>
         </property>
         <property name="textDirection">
          <enum>QProgressBar::TopToBottom</enum>
         </property>
         <property name="format">
          <string>%v/%m PV</string>
         </property>
        </widget>
       </item>
       <item row="4" column="5">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0" rowspan="2">
        <widget class="QLabel" name="img2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1" rowspan="3">
        <widget class="QLabel" name="img3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="4" rowspan="3">
        <widget class="QLabel" name="img1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="gerer_2">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>714</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="10" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Guérir</string>
         </property>
        </widget>
       </item>
       <item row="10" column="5">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>Transférer</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Transférer</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Joueur 1</string>
         </property>
        </widget>
       </item>
       <item row="8" column="7" colspan="2">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Boite PC</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3" colspan="3">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Mon équipe</string>
         </property>
        </widget>
       </item>
       <item row="12" column="7">
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string>Sauvegarder</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" colspan="3">
        <widget class="QComboBox" name="dresseurs1">
         <item>
          <property name="text">
           <string>kigfr&quot;de</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>efgsgs</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Dresseur</string>
         </property>
        </widget>
       </item>
       <item row="7" column="7" colspan="6">
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Hautes herbes</string>
         </property>
        </widget>
       </item>
       <item row="3" column="8">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Niveau:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="6">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="QTableView" name="pc1"/>
       </item>
       <item row="3" column="5">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="8" column="2" rowspan="3">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="11">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Nb points</string>
         </property>
        </widget>
       </item>
       <item row="10" column="7">
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Guérir</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Boite PC</string>
         </property>
        </widget>
       </item>
       <item row="2" column="10" colspan="3">
        <widget class="QComboBox" name="dresseurs2">
         <item>
          <property name="text">
           <string>a</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>b</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="12" column="5">
        <widget class="QPushButton" name="menu_4">
         <property name="text">
          <string>Quitter</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4" colspan="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Capture</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6" rowspan="10">
        <widget class="Line" name="line">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="lineWidth">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Niveau:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Joueur 2</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" colspan="13">
        <widget class="Line" name="line_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="8">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Dresseur</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="6">
        <widget class="QTableView" name="hh1">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="9" rowspan="3">
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="10">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="7" colspan="2">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Hautes herbes</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3" colspan="3">
        <widget class="QTableView" name="equipe1"/>
       </item>
       <item row="9" column="7" colspan="2">
        <widget class="QTableView" name="pc2"/>
       </item>
       <item row="3" column="4">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Nb points</string>
         </property>
        </widget>
       </item>
       <item row="10" column="8">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>Transférer</string>
         </property>
        </widget>
       </item>
       <item row="9" column="11" colspan="2">
        <widget class="QTableView" name="equipe2"/>
       </item>
       <item row="8" column="10" colspan="3">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Mon équipe</string>
         </property>
        </widget>
       </item>
       <item row="10" column="12">
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string>Transférer</string>
         </property>
        </widget>
       </item>
       <item row="3" column="12">
        <widget class="QLabel" name="label_11">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="11" colspan="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Capture</string>
         </property>
        </widget>
       </item>
       <item row="5" column="7" colspan="6">
        <widget class="QTableView" name="hh2">
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>progressBar</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Widget</receiver>
   <slot>test()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>951</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>jouer</sender>
   <signal>pressed()</signal>
   <receiver>Widget</receiver>
   <slot>test()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>908</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gerer</sender>
   <signal>pressed()</signal>
   <receiver>Widget</receiver>
   <slot>test()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>908</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu_2</sender>
   <signal>pressed()</signal>
   <receiver>Widget</receiver>
   <slot>test()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu_3</sender>
   <signal>pressed()</signal>
   <receiver>Widget</receiver>
   <slot>test()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>576</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>571</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menu_4</sender>
   <signal>pressed()</signal>
   <receiver>Widget</receiver>
   <slot>test()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>442</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>test()</slot>
 </slots>
</ui>
